coverage/test/env_test.go

```diff
@@ -17,13 +17,23 @@ limitations under the License.
 package test
 
 import (
-	"testing"
+	"os"
 	"path"
+	"testing"
 )
 
 func Test(t *testing.T) {
 	pathRelToProj := "middle/of/nowhere"
-	expected := path.Join(ProjDir(), pathRelToProj)
+	var projAbsolutePath string
+	testSrcDir := os.Getenv("TEST_SRCDIR")
+	if testSrcDir != "" {
+		projAbsolutePath = path.Join(testSrcDir, projectPathLessTestSrc)
+	}
+
+	gopath := os.Getenv("GOPATH")
+	projAbsolutePath = path.Join(gopath, projectPathLessGoPath)
```

Comment:
You're overriding any value set in line 30, which renders lines 28-31 useless. I suspect there's an `else` misssing here?

Reply:
yes, nice catch. I just updated it